utils/templates/README-module+member.md.in

```diff
@@ -0,0 +1,128 @@
+${name^} module
+============
+
+Description
+-----------
+
+The $name module allows to ensure presence and absence of ${name}s and $name members.
+
+Features
+--------
+
+* ${name^} management
+
+
+Supported FreeIPA Versions
+--------------------------
+
+FreeIPA versions 4.4.0 and up are supported by the ipa$name module.
+
+
+Requirements
+------------
+
+**Controller**
+* Ansible version: 2.8+
+
+**Node**
+* Supported FreeIPA version (see above)
+
+
+Usage
+=====
+
+Example inventory file
+
+```ini
+[ipaserver]
+ipaserver.test.local
+```
+
+
+Example playbook to make sure $name "NAME" is present:
+
+```yaml
+---
+- name: Playbook to manage IPA $name.
+  hosts: ipaserver
+  become: yes
+
+  tasks:
+  - ipa$name:
+      name: NAME
```

Comment:
The `standard password` should be added here: `ipaadmin_password: SomeADMINpassword`.